src/main/java/reposense/parser/RepoConfigCsvParser.java

```diff
@@ -48,23 +48,35 @@ public RepoConfigCsvParser(Path csvFilePath) throws FileNotFoundException {
      * Gets the list of headers that are mandatory for verification.
      */
     @Override
-    protected String[] mandatoryHeaders() {
-        return new String[] {
+    protected String[][] mandatoryHeaders() {
+        String[][] equivalentHeaders = new String[][] {
+                // Can be added in the future
+        };
+
+        String[] singleHeaders = new String[] {
                 LOCATION_HEADER,
         };
+
+        return combineHeaders(equivalentHeaders, singleHeaders);
     }
 
     /**
      * Gets the list of optional headers that can be parsed.
      */
     @Override
-    protected String[] optionalHeaders() {
-        return new String[] {
+    protected String[][] optionalHeaders() {
+        String[][] equivalentHeaders = new String[][] {
+                // Can be added in the future
+        };
+
+        String[] singleHeaders = new String[] {
                 BRANCH_HEADER, FILE_FORMATS_HEADER, IGNORE_GLOB_LIST_HEADER, IGNORE_STANDALONE_CONFIG_HEADER,
                 IGNORE_FILESIZE_LIMIT_HEADER, IGNORE_COMMIT_LIST_CONFIG_HEADER, IGNORE_AUTHOR_LIST_CONFIG_HEADER,
                 SHALLOW_CLONING_CONFIG_HEADER, FIND_PREVIOUS_AUTHORS_CONFIG_HEADER, FILESIZE_LIMIT_HEADER,
                 SKIP_IGNORED_FILE_ANALYSIS_HEADER
         };
+
+        return combineHeaders(equivalentHeaders, singleHeaders);
```

Comment:
Do you think it might be better to place all the single headers into a length 1 array to simplify the process? I.e. something like placing `BRANCH_HEADER` into `{BRANCH_HEADER}` to initialise the `String[][]` directly. 

It avoids the need to treat equivalent and single headers separate and coupling with the `combineHeaders` method. I am aware that that is what `combineHeaders` does but I don't think it should be a necessity to call it like this.

Reply:
I'm not too sure what's the style in reposense. For me, I prefer to keep to minimal changes. Single arrays seems a bit weird to me and that's why I did it this way. But if you think single arrays are ok then sure, I'll make the changes.

Reply:
I personally don't think that there's anything strange with single element arrays or even a need to carefully distinguish the `String[]` groups and the single `String` headers. I would just change the method name to something like `optionalHeaderGroups` and `mandatoryHeaderGroups` instead of `optionalHeaders` and `mandatoryHeaders`. 

I guess my main issue was more of the unused local variables lying around together with the need to always combine them with `combineHeaders`

Reply:
The unused local variables are just there as a template for future dev to know how to use. But sure, I'll just make everything into arrays then.